tests/tests/connectors/test-class-connector-acf.php

```diff
@@ -0,0 +1,193 @@
+<?php
+namespace WP_Stream;
+
+class Test_WP_Stream_Connector_ACF extends WP_StreamTestCase {
+	protected $group_key;
+	protected $field_key;
+
+
+	public function setUp() {
+		parent::setUp();
+
+		// Make partial of Connector_ACF class, with mocked "log" function.
+		$this->mock = $this->getMockBuilder( Connector_ACF::class )
+			->setMethods( [ 'log' ] )
+			->getMock();
+
+		// Register connector.
+		$this->mock->register();
+
+		// Reusable ACF object keys.
+		$this->group_key = __CLASS__;
+		$this->field_key = 'field_5d7812fd000a4';
```

Comment:
@kidunot89 is this field key unique (always expected to be this value) or arbitrary?

Reply:
No, I was using this in an old iteration the test, I remove it.